pkg/synthetics/types.go

```diff
@@ -102,11 +102,11 @@ type SyntheticsMonitorCreateErrorType string
 var SyntheticsMonitorCreateErrorTypeTypes = struct {
 	// Received a request missing required fields or containing invalid data
 	BAD_REQUEST SyntheticsMonitorCreateErrorType
-	// An unknown error occured while processing request to mutate monitor
```

Comment:
Note: The modifications in this file are not related to the issue addressed by the PR, but have been made (spelling mistake corrections) to avoid warnings shown by the linter in this file on all PRs.